fast_forward/interactions_to_itp.py

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Doesn't it make more sense to build the interactions tuples directly in the fitter? Then one only has to clear the interactions dict and add the new ones?

Interactions are defined as NamedTuples in vermouth.molecule. See comment on Fitter.